doc/architecture/trace_file_mcap_format.adoc

```diff
@@ -0,0 +1,121 @@
+** `osi`: SemVer version of the minimum required OSI version
+- Must include metadata with the name `asam_osi` containing at least the following key-value pairs:
+** `zero_time`: ISO 8601 YYYYMMDDThhmmss.f formatted point in time representing the zero time of the scenario
+** `timestamp`: ISO 8601 YYYYMMDDThhmmss.f formatted creation time of the file 
+- It is strongly recommended to include metadata with the name `asam_osi` containing the following key-value pairs:
+** `description`: Short human-readable scenario description
+** `creator`: csv of person or company (not tool) creating the file
+** `license`` csv of spdx identifiers
+** `data_sources` csv of model, scenario player, etc.
+- Additional custom metadata may be added, but it is recommended to add a category with the name `context` where the key represents a prefix and the value pointing to the specification of the metadata. This allows to add other (channel-wise) metadata with the stated prefix. Thus, it becomes clear what a metadata is about and where it is specified. The following examples are given:
+** GAIA-X4PLC-AAD SHACL Shape
+*** Assume you want to embed the hdmap of a scenario in the MCAP file.
+*** The `context` category contains the key `GAIA-X4PLC-AAD-hdmap` with the value `https://github.com/GAIA-X4PLC-AAD/ontology-management-base/blob/main/hdmap/hdmap_shacl.ttl`
+*** A channel metadata contains the key `GAIA-X4PLC-AAD-hdmap` with the value of the hdmap data in the given SHACL shape.
+** openDrive Reference
+*** Assume you want to express that oncoming traffic passes on the right side of the road.
+*** The `context` category contains the key `openDrive` with the value `https://publications.pages.asam.net/standards/ASAM_OpenDRIVE/ASAM_OpenDRIVE_Specification/1.8.1/specification/index.html`
+*** A file metadata in a new metadata category with the arbitrary name `specification` contains the key `openDrive-road-rule` with the value `RHT`
+** Cycle time variation of a sensor
+*** Assume you want to express the interface cycle time variation of a sensor.
+*** The `context` category contains the key `iso_23150` with the value` ISO 23150:2011`
+*** A channel containing `OSI3::SensorData` messages has metadata with the key `iso_23150-cycle-time-variation:` and the value `80`
+
+== Compression
+- OSI-compliant tooling must support compression types: `none`, `lz4`, and `zstd`
+
+
+== Naming Convention
+.mcap files must follow this naming convention:
+
+
+----
+<opt. prefix>_<opt. timestamp>_<type>_<opt. suffix>.mcap
+----
+
+When not using an optional field, the corresponding underscore delimiter must be omitted so that no double underscore is present.
+
+[#tab-MCAP-file-naming-convention]
+.MCAP file naming convention
+[cols="1,1"]
+|===
+|Field |Explanation 
+
+|opt. prefix
+|An optional prefix which may be used to specify the type of scenario (e.g. `cut-in`) or uniqueness of the setup (e.g. `target-5m`). May not contain any `_` characters.
+
+|opt. timestamp
```

Comment:
Maybe we should state that the timestamp can only exceptionally be omitted when there really is no reference to a global time in the file?

I think if you have a real-world capture or any other trace file that has any meaningful relation to a global time frame, it should be visible in the filename.

Reply:
In my opinion the meaningful relation to a global time is the exception here, since simulation should be the "normal" use-case rather than measurements.